client/my-sites/customer-home/cards/actions/quick-links/index.jsx

```diff
@@ -301,12 +299,15 @@ const mapDispatchToProps = {
 	trackDesignLogoAction,
 	addEmailAction,
 	addDomainAction,
+	expand: expandHomeQuickLinks,
+	collapse: collapseHomeQuickLinks,
 };
 
 const mergeProps = ( stateProps, dispatchProps, ownProps ) => {
 	const { editHomePageUrl, isStaticHomePage, siteSlug } = stateProps;
 	return {
 		...stateProps,
+		...dispatchProps,
```

Comment:
I think this was done on purpose originally to not include dispatchProps here. Did we need it?

Reply:
Ah, I see pass through for expand/collapse. 👍